pallets/phala/src/tests.rs

```diff
@@ -634,6 +634,50 @@ fn test_bug_119() {
 	});
 }
 
+/*
+pub fn account<Account: Decode + Default>(name: &'static str, index: u32, seed: u32) -> Account {
+	let entropy = (name, index, seed).using_encoded(sp_io::hashing::blake2_256);
+	Account::decode(&mut &entropy[..]).unwrap_or_default()
+}
+
+#[test]
+fn test_mockdata_transfer_to_chain() {
+	new_test_ext().execute_with(|| {
+		use frame_system::Config;
+
+		let raw_sk = hex::decode("0000000000000000000000000000000000000000000000000000000000000001").unwrap();
+		let sk = ecdsa_load_sk(&raw_sk);
+		let receiver: <Test as Config>::AccountId = account("receiver", 0, 0u32);
+
+		let transfer = Transfer::<<Test as Config>::AccountId, Balance> {
+			dest: receiver,
+			amount: 50u32.into(),
+			sequence: 1,
+		};
+
+		let signature = ecdsa_sign(&sk, &transfer);
+		print!("transfer signature: {:?}", signature);
+	});
+}
+
+#[test]
+fn test_mockdata_sync_worker_message() {
+	new_test_ext().execute_with(|| {
+		let raw_sk = hex::decode("0000000000000000000000000000000000000000000000000000000000000001").unwrap();
+		let sk = ecdsa_load_sk(&raw_sk);
+		let work_message = WorkerMessage {
+			payload: WorkerMessagePayload::Heartbeat{
+				block_num: 123u32,
+				claim_online: true,
+				claim_compute: true,
+			},
+			sequence: 0,
+		};
+		let signature = ecdsa_sign(&sk, &work_message);
+		print!("WorkMessage signature: {:?}", signature);
+	});
+}
+*/
```

Comment:
nit: add a new line here

Reply:
> How much does it cost for `register_worker` and `sync_worker_message`?

Under local dev mode, have a simple test, fee of each:
 - register_worker: 255.5264 micro Unit
 - sync_worker_message: 19.6918 micro Unit